lib/compat/wordpress-6.0/class-wp-webfonts.php

```diff
@@ -78,24 +96,47 @@ public function get_providers() {
 	/**
 	 * Register a webfont.
 	 *
-	 * @param array $font The font arguments.
+	 * @param string $id The unique identifier for the webfont.
+	 * @param array  $font The font arguments.
 	 */
-	public function register_font( $font ) {
+	public function register_font( $id, $font ) {
+		if ( ! $id ) {
+			trigger_error( __( 'An ID is necessary when registering a webfont.', 'gutenberg' ) );
+			return false;
+		}
+
 		$font = $this->validate_font( $font );
 		if ( $font ) {
-			$id                    = $this->get_font_id( $font );
-			self::$webfonts[ $id ] = $font;
+			self::$registered_webfonts[ $id ] = $font;
```

Comment:
Reference the variable as an instance member because that's what it is.
```suggestion
			$this->registered_webfonts[ $id ] = $font;
```